splinterd/src/config/default.rs

```diff
@@ -137,6 +137,27 @@ impl PartialConfigBuilder for DefaultPartialConfigBuilder {
                     level: Some(log::Level::Trace),
                 },
             ),
+            (
+                "sawtooth".to_string(),
```

Comment:
I think scabbard needs to be included in this commit

Reply:
Added a scabbard logger.